Makefile

```diff
@@ -57,6 +57,11 @@ LDFLAGS += -L $(LIBWEBSOCKETS_DIR)/build/lib -lwebsockets
 SRC_DIRS += ${TOP_DIR}/src
 INC_DIRS += ${TOP_DIR}/include
 
+# CURL Requirements
+CURL_DIR = $(TOP_DIR)/framework/curl/curl-8.8.0
+INC_DIRS += $(CURL_DIR)/include
+LDFLAGS += -L $(CURL_DIR)/build/lib -lcurl -Wl,-rpath-link,$(CURL_DIR)/build/lib
```

Comment:
LDFLAGS will come from the .bb file, therefore you should use another variable if you want to set them yourself.

`XLDFLAGS` - maybe?

Reply:
`XLDFLAGS = ${LDFLAGS} -L $(CURL_DIR)/build/lib -lcurl -Wl,-rpath-link,$(CURL_DIR)/build/lib`